design-documents/graph-ql/coverage/company.md

```diff
@@ -0,0 +1,527 @@
+# Queries
+
+```graphql
+type Query {
+    company: Company  @doc(description: "Returns all information about the Company.") 
+    companyUsers (
```

Comment:
All queries with `company` prefix should be grouped under `company` field, instead of being separate queries.

Reply:
fixed